src/app/response-status-bar.component.html

```diff
@@ -1,14 +1,14 @@
 <div class="ms-MessageBar ms-MessageBar-singleline" [ngClass]="message?.backgroundClass" [class.hide-action-bar]="message == null">
     <div class="ms-MessageBar-content">
-        <table>
+        <table aria-labelledby="responseStatusText">
             <tr>
                 <td style="width: 10px;">
                     <div class="ms-MessageBar-icon">
                         <i class="ms-Icon" [ngClass]="message?.icon"></i>
                     </div>
                 </td>
                 <div class="ms-MessageBar-actionables">
-                    <div class="ms-MessageBar-text" tabindex="0" [innerHtml]="messageHTML"></div>
+                    <div id="responseStatusText" class="ms-MessageBar-text" tabindex="0" [innerHtml]="messageHTML" role="alert" aria-live="assertive"></div>
```

Comment:
I'm guessing this won't get read with "polite"?

Reply:
I will check before I merge this pull request. Polite is probably a better choice.